tests/components/litejet/test_scene.py

```diff
@@ -37,8 +36,10 @@ def get_scene_name(number):
         self.mock_lj.get_scene_name.side_effect = get_scene_name
 
         assert setup.setup_component(
-            self.hass, litejet.DOMAIN, {"litejet": {"port": "/tmp/this_will_be_mocked"}}
-        )
+            self.hass, litejet.DOMAIN,
+            {"litejet": {
+                "port": "/tmp/this_will_be_mocked"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Probable insecure usage of temp file/directory.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4620096)